File: 191. Number of 1 Bits/NumberOf1Bits.py
class Solution:
  def hammingWeight(self, n: int) -> int:
    binaryString = "{0:b}".format(n)
    numberOfOnes = 0
    for letter in binaryString:
      if letter == '1':
        numberOfOnes+=1
    return numberOfOnes

s = Solution()
print(s.hammingWeight(3))

File: 191. Number of 1 Bits/NumberOf1Bits.go
package main

import "fmt"

func main() {
	fmt.Println(hammingWeight(7))
}

func hammingWeight(num uint32) int {
	numberOfOnes := 0
	for num > 0 {
		numberOfOnes += int(num & 1)
		num >>= 1
	}
	return numberOfOnes
}
